app/components/content.tsx

```diff
@@ -339,6 +346,52 @@ export default function Content({
                 />
               </div>
             </div>
+
+            <div id="tab-highlight-ai" className="tab-content hidden">
+              <div className="bg-slate-50 p-3 sm:p-4 rounded-md border border-slate-200">
+                <div className="p-4">
+                  <p className="text-sm text-slate-700 mb-4">
+                    Download and setup Highlight AI from{" "}
+                    <a
+                      href="https://highlightai.com"
+                      target="_blank"
+                      rel="noopener noreferrer"
+                      className="text-blue-600 hover:text-blue-800"
+                    >
+                      highlightai.com
+                    </a>
+                  </p>
```

Comment:
In other integrations we don't instruct the users to download the app. I think it may be more helpful to explain how to get to this MCP config modal from within the app (e.g., "Plugins>Custom Plugins>Add SSE...")

Reply:
Yep updating it rn

Reply:
<img width="807" alt="Screenshot 2025-04-16 at 5 26 24 PM" src="https://github.com/user-attachments/assets/b39da220-eda3-45a0-bc8d-fa243ea4cbd1" />
@idosal is this better?

Reply:
@umuthopeyildirim Yeah, looks great! Just to verify - did you create both HighlightAI screenshots?

Reply:
Awesome yes I'm the owner of the images